### ISSUE TITLE
Migrate all layout_* attributes from styles.xml back to layout files

### ISSUE THREAD

**Issue by r4dixx** [r4dixx/VisitCalgary#1]
Because this is a way better practice to do so!

Refer to these two very good articles on the matter for a better understanding:

- [Where do layout attributes belong?](https://tips.seebrock3r.me/where-do-layout-attributes-belong-4d7fd5eaa1fa)
- [Keeping Styles in Scope](http://wiresareobsolete.com/2014/11/keeping-styles-in-scope/)

This small excerpt say it all:

> **Bottom line:** You shouldn’t have to open `styles.xml` to diagnose an issue with the layout XML. You also shouldn’t have to worry about whether a change made in `styles.xml` to fix a specific layout problem is going to propagate to other layouts, possibly breaking something elsewhere, since “there must have been a reason to abstract it to a common location”.

And this is just one of the many reasons.

Also, as we're into `styles.xml`, this might be a good idea to clean it up a little bit as most of the style attributes there won't be re-used anywhere else 

So in short, don't **blindly put everything into `styles.xml`**:
- Leave layout_* attributes to their respective layout files
- Only put attributes there if they will be re-used elsewhere
